CSharp_Course_Project/Program.cs

```diff
@@ -4,9 +4,442 @@ namespace CSharp_Course_Project
+        {
+            string n = Console.ReadLine();
+            int number = Convert.ToInt32(n);
+
+            if (n.Length == 3 && number % 2 != 0)
+            {
+                Console.WriteLine($"{number} is odd and 3-digit number");
+            }
+            else
+            {
+                Console.WriteLine($"{number} is not odd and 3-digit number");
+            }
+        }
+
+        public static void ThreeNumbersOperations()
+        {
+            int x = 10, y = 12, z = 3;
+            x += y - x++ * z; // 10 + 12 - (10 * 3)
+            Console.WriteLine(x); //-8
+            z = --x - y * 5; // -9 - (12 * 5)
+            Console.WriteLine(z);
+            y /= x + 5 % z;
+            Console.WriteLine(y);
+            z = x++ + y * 5;
+            Console.WriteLine(z);
+            x = y - x++ * z;
+            Console.WriteLine(x);
+        }
+
+        public static void ConsoleCalculator()
+        {
+            while (true)
+            {
+                Console.Write("Enter the first number: ");
+                string op1 = Console.ReadLine();
+                bool isFirstNumber = Int32.TryParse(op1, out int operand1);
+                Console.Write("Enter the second number: ");
+                string op2 = Console.ReadLine();
+                bool isSecondNumber = Int32.TryParse(op2, out int operand2);
+                Console.Write("Enter the operation symbol: ");
+                string symbol = Console.ReadLine();
+
+                if (!isFirstNumber || !isSecondNumber)
+                {
+                    Console.WriteLine("Entered numbers are not numberic values. Try again: ");
+                    continue;
+                }
+                else if (operand2 == 0 && symbol == "/")
+                {
+                    Console.WriteLine("Сannot divide by 0. Try again:");
+                    continue;
+                }
+                else
+                {
+                    int result = 0;
+
+                    switch (symbol)
+                    {
+                        case "+":
+                            result = operand1 + operand2;
+                            break;
+                        case "-":
+                            result = operand1 - operand2;
+                            break;
+                        case "*":
+                            result = operand1 * operand2;
+                            break;
+                        case "/":
+                            result = operand1 / operand2;
+                            break;
+                        default:
+                            Console.WriteLine("Wrong operation symbol. Try again:");
+                            continue;
+                    }
+                    Console.WriteLine(result);
+                }
+                Console.WriteLine("Do you want to proceed? Y/N");
+                string proceed = Console.ReadLine();
+                if (proceed == "N" || proceed == "n")
```

Comment:
Here is a cheat - you can low all chars in string and then compare to lower one:
proceed.ToLower() == "n"